route.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const data = require('./Data.json')
+const route = express()
+route.use(express.json())
+
+route.get('/',(req,res)=>{
+    res.send(data)
+})
+
+route.post('/',(req,res)=>{
```

Comment:
Add error handling to CRUD operations. This can be done by wrapping the operations in try-catch blocks and sending an appropriate response in case of an error. [important]